content/lessons/chapter-4/public-key-3.tsx

```diff
@@ -157,7 +157,7 @@ def privatekey_to_publickey(private_key):
           {t(`chapter_four.public_key_three.${language}.paragraph_three`)}
         </Text>
 
-        <Accordion
+        <Accordion className="mt-4"
```

Comment:
Pretty sure you'll need to add a corresponding param to accept this as `Accordian` is a handrolled component not any default element.

Reply:
Ooof. Sorry. That was me. I thought I could sneak a quick change in before the merge, hah. I'll look for examples in the code that do what you described. Thanks for diagnosing this so quickly!

Reply:
On it 👀